### PR TITLE
Added npm/bower to Insights configuration

### PR DESCRIPTION
**Pull request by clintonb**
@e0d

### CODE REVIEW

**Comment by e0d** [comment]
:+1:

File: playbooks/roles/insights/defaults/main.yml [hunk 1]
```diff
@@ -120,6 +120,10 @@ insights_python_path: "{{ insights_code_dir }}/analytics_dashboard"
 insights_conf_dir: "{{ insights_home }}"
 insights_log_dir: "{{ COMMON_LOG_DIR }}/{{ insights_service_name }}"
 
+insights_nodeenv_dir: "{{ insights_home }}/nodeenevs/{{ insights_service_name }}"
```
Comment (feanil) [inline playbooks/roles/insights/defaults/main.yml:123]: 'nodeenevs' => 'nodeenvs'

Comment (clintonb) [inline playbooks/roles/insights/defaults/main.yml:123]: Good thing we have variables. Will fix in a separate PR.

Comment (clintonb) [inline playbooks/roles/insights/defaults/main.yml:123]: See #1638.

**Pull request merged**